brewtils/request_handling.py

```diff
@@ -383,25 +383,25 @@ def __init__(self, ez_client, shutdown_event, **kwargs):
         self.max_timeout = kwargs.get("max_timeout", 30)
         self.starting_timeout = kwargs.get("starting_timeout", 5)
 
-        # Tightly manage when we're in an 'error' state, aka Brew-view is down
-        self.brew_view_error_condition = threading.Condition()
-        self.brew_view_down = False
+        # Tightly manage when we're in an 'error' state, aka brewtils is down
```

Comment:
These brew-view --> brewtils renames should all be brew-view --> beergarden.  It's beergarden being down that this is handling.  Brewtils is just this library, and thus cannot really be up or down.